scopes/harmony/aspect-loader/plugins.ts

```diff
@@ -98,12 +101,14 @@ export class Plugins {
     triggerOnAspectLoadError: OnAspectLoadErrorHandler,
     logger: Logger,
     resolvePath?: (path: string) => string
-  ) {
+  ): Plugins {
     const plugins = defs.flatMap((pluginDef) => {
-      const files =
-        typeof pluginDef.pattern === 'string'
-          ? component.filesystem.byGlob([pluginDef.pattern])
-          : component.filesystem.byRegex(pluginDef.pattern);
+      const files = this.getFileMatches(component, pluginDef);
+
+      if (files.length === 0 && !Plugins.checkedComponents.has(component.id.toString())) {
+        logger.consoleWarning(this.constructWarningMessage(component));
```

Comment:
Most components don't have plugins inside.
Also many aspects are not plugins.
So we need to make sure we don't show that error when not necessary

Reply:
not sure how a regular component or an aspect will get to this check